lib/hammer_cli_foreman/personal_access_token.rb

```diff
@@ -0,0 +1,48 @@
+module HammerCLIForeman
+  class PersonalAccessToken < HammerCLIForeman::Command
+    resource :personal_access_tokens
+    command_name "access-token"
+    desc _("Manage personal access tokens")
+
+    class ListCommand < HammerCLIForeman::ListCommand
+      output do
+        field :id, _("Id")
+        field :name, _("Name")
+        field :active?, _("Active"), Fields::Boolean
+        field :expires_at, _("Expires at"), Fields::Date
+      end
+
+      build_options
+    end
+
+    class CreateCommand < HammerCLIForeman::CreateCommand
+      success_message _("Personal access token [%{name}] created.")
+      failure_message _("Could not create personal access token [%{name}]")
+
+      output do
+        field :token_value, _("Token value")
+      end
```

Comment:
Could you move this to the success message so that the value is on a separate line and on the last line of the output? It will be much easier to grep it then (just `tail -1`).